index.html

```diff
@@ -4753,6 +4761,94 @@ <h2><span class="chunk">sPLT</span>
 <p>Multiple <span class="chunk">sPLT</span> chunks are permitted,
 but each shall have a different palette name.</p>
 </section>
+
+  <!-- Maintain a fragment named "11eXIf" to preserve incoming links to it -->
+<section id="11eXIf">
+  <h2><span class="chunk">eXIf</span>
+    Exchangeable Image File (Exif) Profile</h2>
+
+  <p>The four-byte chunk type field contains the decimal values</p>
+
+  <pre>
+  101 88 73 102
+  </pre>
+
+  <p>The data segment of the <span class="chunk">eXIf</span> chunk
+    contains an Exif profile in the format specified in
+    "4.7.2 Interoperability Structure of APP1 in Compressed Data"
+    of [[CIPA DC-008-2016]]
```

Comment:
This looks similar to ReSpec formatting for creating a link. But I thought that was [[=name=]].
Is this a link?
(I feel like it would be nice for this to be a link, if it isn't.)

Reply:
The syntax is correct. It is a [bibliographical reference](https://respec.org/docs/#references), except that [specref](https://github.com/tobie/specref) doesn't know about this particular one so it shows up in the references section with an error message.

I now fixed this (and updated the spec to 2019, the older spec is 404)